lib/RssBridge.php

```diff
@@ -16,6 +16,8 @@
 require __DIR__ . '/Cache.php';
 require __DIR__ . '/Authentication.php';
 require __DIR__ . '/Configuration.php';
+require __DIR__ . '/BridgeList.php';
```

Comment:
That's really nitpicking, but as BridgeList depends of BridgeCard, it would be better to move it before.